src/main/java/Duke.java

```diff
@@ -1,10 +1,312 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.Files;
+
+import java.util.Scanner;
+import java.util.List;
+import java.util.ArrayList;
+
 public class Duke {
+    private final String VERSION_NUMBER = "1.0.0";
+    private final Scanner SC = new Scanner(System.in);
+    private final Path DUKE_DATA_FILE_PATH = Paths.get("data", "duke.txt");
+    private final Path DUKE_DATA_DIR_PATH = Paths.get("data");
+    private final String NEW_LINE = "\n";
+    private final String HORIZONTAL_LINE =
+            "    ____________________________________________________________";
+    private final String PADDING = "      ";
+
+    private enum Commands {
+        EXIT("BYE"), DELETE("DELETE "), LIST("LIST"),
+        DONE("DONE "), TODO("TODO "), EVENT("EVENT "),
+        DEADLINE("DEADLINE "), DELETEALL("DELETE ALL");
+        private final String str;
+        Commands(String str){
+            this.str = str;
+        }
+        public String getString() {
+            return this.str;
+        }
+    }
+
+    private final String MESSAGE_TEMPLATE = HORIZONTAL_LINE + NEW_LINE + PADDING + "%s"
+            + NEW_LINE + HORIZONTAL_LINE + NEW_LINE + NEW_LINE;
+    private final String MESSAGE_TEMPLATE_VERBAL = HORIZONTAL_LINE + NEW_LINE + PADDING + "Deuk: %s"
+            + NEW_LINE + HORIZONTAL_LINE + NEW_LINE + NEW_LINE;
+    private final String MESSAGE_TEMPLATE_ERROR = HORIZONTAL_LINE + NEW_LINE + PADDING
+            + "☹ OOPS!!! %s"
+            + NEW_LINE + HORIZONTAL_LINE + NEW_LINE + NEW_LINE;
+    private final List<Task> storageList = new ArrayList<>();
+
+    public void init() {
+        sayHello();
+        try {
+            loadTasksFromDisk();
+        }
+        catch (DukeDataFolderException ex) {
+            printError(ex.getMessage());
+            File dir = new File(DUKE_DATA_DIR_PATH.toUri());
+            Boolean isCreated = dir.mkdir();
+            if (isCreated) {
+                echoBack("Successfully created Deuk Data Folder");
+                try {
+                    FileWriter fw = new FileWriter(DUKE_DATA_FILE_PATH.toString());
+                    fw.close();
+                }
+                catch (IOException err) {
+                    printError(err.getMessage());
+                }
+            }
+            else {
+                printError("Failed to create Deuk Data Folder");
+            }
+        }
+        catch (DukeException ex) {
+            printError(ex.getMessage());
+        }
+        catch (FileNotFoundException ex) {
+            printError("Missing Deuk Data File!" + NEW_LINE + PADDING +
+                "Creating new Deuk Data File..."
```

Comment:
Just a very minor detail here, I think you can consider put the `+` in the beginning of the new line instead of putting it at the end of line 72 :D